packages/compiler-sfc/src/parse.ts

```diff
@@ -47,6 +47,7 @@ export interface SFCDescriptor {
   customBlocks: SFCBlock[]
 }
 
+const sourceToSFC = new Map<string, SFCDescriptor>()
```

Comment:
The cache should be using [lru-cache](https://github.com/vuejs/vue-next/blob/master/packages/compiler-sfc/package.json#L33) to avoid forever growing memory.

Reply:
@yyx990803  Yep, that’s necessary. I will improve it later. Maybe this sunday because I forget to take my pc.